shared/src/main/scala/com/raquo/domtypes/defs/reflectedAttrs/ReflectedHtmlAttrDefs.scala

```diff
@@ -1105,6 +1105,27 @@ object ReflectedHtmlAttrDefs {
       docUrls = Nil,
     ),
 
+    ReflectedHtmlAttrDef(
+      scalaName = "crossorigin",
+      domAttrName = "crossorigin",
+      domPropName = "crossorigin",
+      scalaValueType = "String",
+      domPropValueType = "String",
+      attrCodec = "StringAsIs",
+      propCodec = "StringAsIs",
+      commentLines = List(
+        "This attribute specifies how the browser should handle cross-origin",
+        "requests for the associated resource. It controls whether the resource can",
+        "be loaded when requested from a different domain, and how to handle potential",
+        "security issues, like CORS (Cross-Origin Resource Sharing) policies.",
+        "The value of this attribute determines whether the browser will allow or",
+        "block loading of the resource, helping to enhance web security."
```

Comment:
Please add two more lines to the docs:
"",
"""Allowed values: "anonymous" | "use-credentials" | "" (same as "anonymous") """

Reply:
fixed